guardian_linter_watchdog.py

```
@@ -21,43 +20,58 @@
 logger = logging.getLogger(__name__)
 
+
 async def run_linter(file_path: Path):
     """Run the linter on a specific file."""
     if not any(part in EXCLUDED_DIRS for part in file_path.parts):
         command = LINT_COMMAND + [str(file_path)]
-        logger.info(f"Guardian: Analyzing {file_path.relative_to(PROJECT_ROOT)}...")
-        
+        logger.info(
+            f"Guardian: Analyzing {
+                file_path.relative_to(PROJECT_ROOT)}..."
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)